### DISCUSSION TITLE
Create user-maintained lists of community plugins

### DISCUSSION THREAD

**Discussion by HEmile** [HEmile/obsidian-api-docs#3] [Ideas]
Like how the Obsidian help vault has a directory for core plugins, we could also have one for community plugins. This would allow linking to these plugins, and creating lists to help people find the right plugin for their problem.

Do you think this is in scope? Or should this vault focus on the API?

**Comment by urishiraval** [comment]
Interesting, I'm down for that

**Comment by tgrosinger** [comment]
I think that could be useful, especially if setup in a way that links to plugins which implement particular parts of the API. For example, when I wanted to learn how to create a custom View, I looked through Github to find a plugin which was already using it to take inspiration from. Similarly, when I wanted to use Svelte, I searched to find a plugin which was using it and copied parts of their config.

**Comment by argenos** [comment]
I like this idea. This is probably useful in a "Getting started" kind of way.  There's also this question in [the forum](https://forum.obsidian.md/t/how-to-get-started-with-developing-a-custom-plugin/8157/) (although perhaps that should be a separate discussion?)
